docs/interfaces.md

```diff
@@ -199,4 +199,11 @@ Auto-Connect: no
 Can query hardware information from the system.
 
 Usage: reserved
-Auto-Connect: no
\ No newline at end of file
+Auto-Connect: no
+
+### fuse
+
+Lets you run a fuse filesystem, accessing /dev/fuse and /sys/fs/fuse/connections.
```

Comment:
This should use language that is similar to the other entries. I don't particularly like either style ("you", or "Can foo"), but consistency wins.